Cargo.toml

```diff
@@ -55,7 +43,6 @@ moss_base = { path = "lib/moss_base" }
 
 base = { path = "platform/base" }
 configuration = { path = "platform/configuration" }
-workbench = { path = "platform/workbench" }
```

Comment:
This module is also needed, maybe I deleted it when changing versions, but we need it